build-system/cpp-cmake-conan/src/install_deps.py

```diff
@@ -30,7 +30,13 @@ def safe_get_workspace_dir() -> str:
     try:
         return get_workspace_dir()
     except Exception:
-        return "."
+        return os.path.abspath(".")
+
+
+def get_build_folder(build_arch: str, host_arch: str):
+    if host_arch == build_arch:
+        return os.path.join(safe_get_workspace_dir(), "build")
+    return os.path.join(safe_get_workspace_dir(), f"build_linux_{host_arch}")
```

Comment:
why not always use that?

Reply:
Because some of our simulation tooling is expecting things in the "simple" build folder.

Reply:
It's not about a single file (or a few of them). It's about keeping all the build artefacts for the different architectures separated. 
I mean, I'd also prefer naming the folders equally. I currently cannot oversee the effort to change away from the simple build folder. So, I just thought this is a first improvement for not mixing up things in a single build folder

Reply:
we should think about having the following structure: `build/linux_<targetarch>/...` which other toolchain components rely on the build dir? the runtime?

Reply:
discussed that yesterday briefly with @BjoernAtBosch and came to the conclusion that this is the correct way but we wanted to have this PR as intermediate solution for now. 

I would also say there is not much dependency to the build folder, but not checked and before it gets to much of refactoring, lets do it after friday...